HW_2/src/components/UserInfoTable.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import "./UserInfoTable.css";
+import BackButton from "./BackButton.js"
+
+
+export default class UserInfoTable extends React.Component{
+	state = {
+		user: []
+	}
+	componentDidMount(){
+		fetch(`https://jsonplaceholder.typicode.com/users/${this.props.idUser}`)
```

Comment:
don't use fetch directly in component, this is repository-specific logic. What if this request will be used in more than 2 places, what will you do? copy code? move this code to another file